app/scripts/head/startup-styles.js

```diff
@@ -8,8 +8,6 @@
 
 // NOTE: This file must run in IE8+, so no ES5/ES6 features!
 
-//jshint es3: true, es5: false, esnext: false
```

Comment:
There doesnt seem to be a way for this. There is a switch for es5/6 but not for es3.